apps/ejabberd/test/cassandra_SUITE.erl

```diff
@@ -0,0 +1,93 @@
+%%==============================================================================
+%% Copyright 2016 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%==============================================================================
+-module(cassandra_SUITE).
+-compile(export_all).
+
+-include_lib("common_test/include/ct.hrl").
+
+%%--------------------------------------------------------------------
+%% Suite configuration
+%%--------------------------------------------------------------------
+
+all() ->
+    [{group, cassandra_group}].
+
+groups() ->
+     [{cassandra_group, [sequence], test_cases()}].
+
+test_cases() -> [check_connection].
+
+suite() ->
+    [].
+
+init_per_suite(Config) ->
+    Config.
+
+end_per_suite(Config) ->
+    Config.
+
+init_per_group(_GroupName, Config) ->
+    Config.
+
+end_per_group(_GroupName, Config) ->
+    Config.
+
+init_per_testcase(_CaseName, Config) ->
+    Config.
+
+end_per_testcase(_CaseName, Config) ->
+    Config.
+
+%%--------------------------------------------------------------------
+%% Last tests
+%%--------------------------------------------------------------------
+check_connection(Config) ->
+    case gen_tcp:connect("127.0.0.1", 9042, []) of
+        {ok, _} ->
+            ct:pal("Cassandra is listening on port 9042", []),
+            check_connection2(Config);
+        _ ->
+            ct:pal("Cassandra is not listening on port 9042, skip the test", []),
```

Comment:
I suggest to return a comment from the test case instead of or in addition to the log message.